src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleLoggerProvider.cs

```diff
@@ -13,6 +13,7 @@ namespace Microsoft.Extensions.Logging.Console
     /// <summary>
     /// A provider of <see cref="ConsoleLogger"/> instances.
     /// </summary>
+    [UnsupportedOSPlatform("android")]
```

Comment:
Why is it not supported? There does not seem to be anything notworking?

Reply:
Yeah, this annotation seems to be irrelevant, Alexander suggested moving the existing `!RuntimeInformation.IsOSPlatform(OSPlatform.Windows)` check from the private `DoesConsoleSupportAnsi()` method to the public ctor.